app/javascript/react/screens/App/common/reactLoadable.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Loadable from 'react-loadable';
+
+const LoadingPage = ({ error, pastDelay, timedOut }) => {
+  if (error) {
+    return <div>Error! Please refresh and try again.</div>;
+  } else if (timedOut) {
```

Comment:
i18n support on these messages? Curious what @serenamarie125 and @vconzola what to display here if the entire page fails loading...